app/templates/road-sign-concepts/road-sign-concept.hbs

```diff
@@ -1,78 +1,168 @@
+<AuDataTable @content={{this.model.subSigns}} @fields="Image Meaning Code"
+  @noDataMessage={{t "roadSignConcept.crud.noData" }} @size={{this.size}} @page={{this.page}} @sort={{this.sort}} as
+  |s|>
+  <s.menu as |menu|>
+    <menu.general>
+      <AuToolbar class="au-o-box">
+        <AuToolbarGroup>
+          <AuHeading @skin="2">
+            {{t "roadSignConcept.subSign.name" }}
+          </AuHeading>
+        </AuToolbarGroup>
+      </AuToolbar>
+    </menu.general>
+  </s.menu>
+  <s.content as |c|>
+    <c.header>
+      <th class="data-table__header-title">
+        {{t "roadSignConcept.attr.code" }}
+      </th>
+      <th class="data-table__header-title">
+        {{t "roadSignConcept.attr.imageHeader"}}
+      </th>
+      <th class="data-table__header-title">
+        {{t "roadSignConcept.attr.meaning"}}
+      </th>
+      <th class="data-table__header-title"></th>
+    </c.header>
+    <c.body as |subSign|>
+      <td>
+        {{subSign.roadSignConceptCode}}
+      </td>
+      <td>
+        <img src={{subSign.image}} alt="" role="presentation" />
+      </td>
+      <td>
+        {{subSign.meaning}}
+      </td>
+      <td>
+        <AuButton @alert={{true}} {{on "click" (fn this.removeSubSign subSign)}}>
+          <AuIcon @icon="bin" @alignment="center" @size="large" @ariaHidden={{true}} />
+        </AuButton>
+      </td>
+    </c.body>
+  </s.content>
+</AuDataTable>
+
+<AuDataTable @content={{this.model.relatedRoadSignConcepts}} @fields="Image Meaning Code"
+  @noDataMessage={{t "roadSignConcept.crud.noData" }} @size={{this.size}} @page={{this.page}} @sort={{this.sort}} as
+  |s|>
+  <s.menu as |menu|>
+    <menu.general>
+      <AuToolbar class="au-o-box">
+        <AuToolbarGroup>
+          <AuHeading @skin="2">
+            {{t "roadSignConcept.relatedSign.name" }}
+          </AuHeading>
+        </AuToolbarGroup>
+      </AuToolbar>
+    </menu.general>
+  </s.menu>
+  <s.content as |c|>
+    <c.header>
+      <th class="data-table__header-title">
+        {{t "roadSignConcept.attr.code" }}
+      </th>
+      <th class="data-table__header-title">
+        {{t "roadSignConcept.attr.imageHeader"}}
+      </th>
+      <th class="data-table__header-title">
+        {{t "roadSignConcept.attr.meaning"}}
+      </th>
+      <th class="data-table__header-title"></th>
+    </c.header>
+    <c.body as |relatedRoadSignConcept|>
+      <td>
+        {{relatedRoadSignConcept.roadSignConceptCode}}
+      </td>
+      <td>
+        <img src={{relatedRoadSignConcept.image}} alt="" role="presentation" />
+      </td>
+      <td>
+        {{relatedRoadSignConcept.meaning}}
+      </td>
+      <td>
+        <AuButton @alert={{true}} {{on "click" (fn this.removeRelatedRoadSignConcept relatedRoadSignConcept)}}>
+          <AuIcon @icon="bin" @alignment="center" @size="large" @ariaHidden={{true}} />
+        </AuButton>
+      </td>
+    </c.body>
+  </s.content>
+</AuDataTable>
+
+
+<AuToolbar @border="top" @size="large">
```

Comment:
I think moving the buttons up would make more sense in this layout. They are kind of hard to find with the tables above them. I think Anita had a version where the buttons where moved to the top?